crates/daemon/src/lib.rs

```diff
@@ -445,14 +453,46 @@ where
         self.wallet_actor.send(wallet::Sync).await?;
         Ok(())
     }
+
+    #[instrument(skip(self, seed), err)]
+    pub async fn import_seed(
+        &self,
+        seed: Vec<u8>,
+        data_dir: PathBuf,
+        network: Network,
+    ) -> Result<()> {
+        let open_cfd_ids = self.db.load_open_cfd_ids().await?;
+
+        if !open_cfd_ids.is_empty() {
+            tracing::warn!("Rejecting imported seed as open CFDs exist.");
+            // in case of open cfds we do not accept an imported seed.
+            return Err(anyhow!(
+                "Can not accept imported seed since open CFDs exist."
+            ));
```

Comment:
🔧 Logging and bubbling up at the same time seems unnecessary. I would just bubble up the error and let the caller decide what to do (probably log it).

The idiomatic way to do this with `anyhow` would be:

```suggestion
            use anyhow::bail;
            
            bail!("Cannot accept imported seed since open CFDs exist");
```

I would also omit the comment since it's saying basically the same as the message and the log.